guide/payments/request.md

```diff
@@ -222,28 +232,36 @@ Contact cards are a great way to  store, manage and share reusable payment reque
    height = 541
 %}
 
-{% include picture.html
-   image = "/assets/images/guide/payments/request/contact-list.png"
-   retina = "/assets/images/guide/payments/request/contact-list@2x.png"
-   layout = "shadow"
-   caption = "Contact lists are great for storing and managing reusable payment requests."
-   alt-text = "Screen showing contact list."
-   width = 250
-   height = 541
-%}
+</div>
+
+### Node ID
+
+A users Lightning node ID, which every non-custodial Lightning wallet has, can be a reusable way to receive Lightning payments using [Keysends](https://bitcoinops.org/en/topics/spontaneous-payments/).
+
+<div class="center" markdown="1">
 
 {% include picture.html
-   image = "/assets/images/guide/payments/request/contact-card.png"
-   retina = "/assets/images/guide/payments/request/contact-card@2x.png"
-   layout = "shadow"
-   caption = "Contact cards can store reusable payment requests of other users too."
-   alt-text = "Screen showing another users contact card."
+   image = "/assets/images/guide/payments/request/node-id.png"
+   retina = "/assets/images/guide/payments/request/node-id@2x.png"
+   layout = "float-left-desktop -background -shadow"
+   caption = "Node ID's are a convenient, reusable way to receive Keysend payments."
+   alt-text = "Screen showing single payment request settings."
    width = 250
    height = 541
 %}
 
+If a sender supports Keysends they can make recurring payments with attached messages to a users node ID without an invoice.
+
+Having the option to quickly generate a personalised QR code of your node ID from the home screen is recommended.
+
+This form of reusable payment request is not widely supported but offer a unique, way for users to make payments, tip users, and send messages over the Lightning network.
+
 </div>
 
+Several not widely supported reusable payments request formats exist. For Lightning, there are [BOLT12 offers](https://bolt12.org/) and AMP invoices.
```

Comment:
Since there is no header to this paragraph, I thought I was still reading about Node Ids. Why not put this further up with the intro the "Reusable payment requests". Could probably also shorten it and link to the Payment request formats page for all the details.